VTUL/dlp-access
# Change metadata fields

**JIRA Ticket**: (link) (:star:)
https://webapps.es.vt.edu/jira/browse/LIBTD-2665

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)
DLP metadata workgroup spreadsheet - https://docs.google.com/spreadsheets/d/1UK1ekAsmtj5KyLeKFv-EAhgl-xSBQeGQ6bXViR0Myag/edit#gid=632902402

# What does this Pull Request do? (:star:)
-Deletes unneeded elements: circa - this info should go into the display_date field
-Make all fields multivalued except for Identifier, title, start date, end date (and boolean fields): changes bibliographic_citation, display_date, extent, rights_holder, rights_statement
-Rename fields: belongs_to to is_part_of, related_url to relation, location to spatial, rights_statement to rights, reference to references, resource_type to type
-Also makes the necessary corrections in the app and tests to accommodate these changes.

# What's the changes? (:star:)
-updates a bunch of files for the GraphQL
-updates 5 cypress tests
-updates a bunch of files in an "examples" directory. I'm not even sure what this is or if we use it anymore.
-MetadataRenderer - makes some corrections for how to display the metadata in addition to just updating the element names where needed.
-SiteAdmin - includes a small fix to make the group names lowercase before comparing to site's repo type 
-updates a bunch of other components which just needed an element name updated here and there.

# How should this be tested?
-The parts of the site which display or consume metadata should work as they did before - search page, archive page, collection browse page, collection page, siteAdmin (Search page config, browse collections pages, displayed attributes, new/update item, new/update collection, CSV export)

branch is LIBTD-2665

**Note - this PR requires using the metaupdate backend.
**Additional Note - After this PR is merged we will need to migrate the current archive and collection tables to match the new data formats. Two lambdas are available in the AWS pprd account to accomplish this. 1) multivalues - converts data for a metadata element (specified in an env variable) from string to array (empty strings become null), 2) changeAttrName - copies data from one metadata element to another with a new name. We will also need to update the site tables manually.

# Interested parties
@whunter 

(:star:) Required fields
